src/main/resources/application.yaml

```diff
@@ -101,7 +103,8 @@ spring:
     mode: TEXT
     suffix: .sql
 
-  jersey.application-path: rest
+  # this must be `camunda` to match module descriptor path patterns
+  jersey.application-path: camunda
```

Comment:
This will eventually need to become `workflow-engine` as per:
- https://folio-org.atlassian.net/browse/MODWRKFLOW-15
- https://folio-org.atlassian.net/browse/MODCAMUNDA-4

This is something for some future sprint to be aware of.